CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Changelog
 ### next
+  * tpm2_import: support specifying parent key with a context file,
+    --parent-key-handle/-H becomes --parent-key-context/-C
```

Comment:
We should probably do `--parent-key`, since it can be either a context or a handle.

Reply:
agreed, will change.